### DISCUSSION TITLE
Detect incorrect files in season folder

### DISCUSSION THREAD

**Discussion by wimpyrbx** [pymedusa/Medusa#5792] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
nope

**Describe the solution you'd like**
on/off (default off i would guess) setting to be able to note if a show has episodes that are not inside their respective "season xx" folders.

example: if Showname.S01E01.1080p.Bluray.mkv is located for some reason inside Season 02 folder instead of Season 01
OR
example: Showname.S01E01.1080p.Bluray.mkv is located in Season 01/Showname.S01E01.1080p.Bluray/ folder for some reason. (manual extract etc)

Basically if episode sXXeYY is not located directly inside their respective Season XX folder.

These could be shown in the lists with a different colored background or an icon (if the option is set to ON in settings).

I can provide photoshopped screenshot if this post is unclear.

**Additional context**
To give users who might have included shows into Medusa that has existing files that are not "up to standards" in regards to filename/folders.

Basically a on/off switch for those users who wish to get hints on how to clean up their collection to adhere to "Season XX" setup.

**Comment by wimpyrbx** [comment]
Example 3:
if for some reason Showname.S01E01.1080p.Bluray.mkv is located in "Series 1" foler while other episode from same season is in "Season 01".

**Comment by wimpyrbx** [comment]
Adding to this feature request.
This does not need to be related to any udpates in database or whatever.
Basically a way to identify such occurences when rendering displayShow page.

**Comment by wimpyrbx** [comment]
![image](https://user-images.githubusercontent.com/13415240/49376701-6c164d00-f708-11e8-9d89-0094cda683ca.png)

Photoshopped example

**Comment by wimpyrbx** [comment]
I would believe it would be a simple IF statement in the rendering code when building the tables.
`
IF episode.location <> settings.seasonformat THEN <img src='folder-alert-icon.png'>
`
yeah I obviously have no idea what the entities are named but you get the idea.

**Comment by wimpyrbx** [comment]
bump? :P

**Comment by p0psicles** [comment]
Why would you put an episode in the wrong season folder? Afaik this isn't a bug

**Comment by wimpyrbx** [comment]
@p0psicles it's not a bug.
Just thought it would not to be too hard, but would be nice to offer as an option for those individuals who might have some manually added (or existing) tvshows into the mix that were not up to the folder/file-structure "standard".

Disregard this if it is not in the scope.
